docs/start/install.mdx

```diff
@@ -137,6 +137,121 @@ Additionally, Qiskit only supports CPython. Running with other Python interprete
   - Windows 32 bit for Python 3.10 or later
 </details>
 
+## Qiskit Versioning
+
+Qiskit version numbers follow [Semantic Versioning](https://semver.org/).
+The version number is comprised of 3 primary components, the major, minor, and
+patch versions. For a version number `X.Y.Z` where `X` is the major version,
+`Y` is the minor version, and `Z` is the patch version.
+
+Breaking API changes are reserved for major version releases. The **minimum**
+period between major version releases is one year. Minor versions will be
+periodically (currently every three months) published for the current major
+version which add new features and bug fixes. For the most recent minor version,
+there will also be new patch versions published as bugs are identified and fixed
+on that release series. A tentative release schedule is included below:
+
+![](/images/start/install/release_schedule.png)
+
+For an up-to-date release schedule you should refer to the Qiskit Github project's
+[milestones list](https://github.com/Qiskit/qiskit/milestones), which will always
+contain the current release plan.
+
+With the release of a new major version, the previous major version is supported
+for at least 6 months; only bug and security fixes will be accepted during this
+time and only patch releases will be published for this major version. A final
+patch version will be published when support is dropped and that release will
+also document the end of support for that major version series. A longer
```

Comment:
Can you clarify what this means to document this? Does it include deprecation warning logic?

Reply:
I just meant something like the prelude in the release notes will say something like "0.46.4 is the final release for the major version 0.x and there will be no further release for the 0.x release series."